spectator/publisher.h

```diff
@@ -34,6 +35,8 @@ class SpectatordPublisher {
   asio::local::datagram_protocol::socket local_socket_;
   std::string buffer_;
   uint32_t bytes_to_buffer_;
+  std::chrono::steady_clock::time_point last_flush_time_;
+  std::chrono::seconds flush_interval_;
```

Comment:
I think we should have the flush interval be mills just for additional granularity.